contracts/middle-layer/resource-mirror/CmnHub.sol

```diff
@@ -169,6 +170,7 @@ abstract contract CmnHub is CmnStorage, Initializable, ICmnHub, IMiddleLayer {
                 bytes memory reason;
                 bool failed;
                 uint256 gasBefore = gasleft();
+                require(gasBefore > callbackGasLimit, "insufficient gas");
                 try
                     IApplication(extraData.appAddress).greenfieldCall{ gas: callbackGasLimit }(
```

Comment:
I think there is no need for this checker, if gas is not enough, the whole tx will be revert and cause no impact.

Reply:
It is under a whole try catch in the CrossChain contract, won't be reverted